Tools/BuildAndPackage.ps1

```diff
@@ -249,11 +249,16 @@ function ConfirmExists($relativePath) {
    }
 }
 
+# Move BootstrapCloudToolsForPowerShell.ps1 to the GoogleCloud folder as this is
+# where "Cloud Tools For PowerShell" program in the Cloud SDK is looking for it.
+Move-Item "$googleCloudDir\BootstrapCloudToolsForPowerShell.ps1" `
+          "$powerShellDir\GoogleCloud\BootstrapCloudToolsForPowerShell.ps1"
+
 ConfirmExists "PowerShell\GoogleCloud\$normalizedVersion\GoogleCloud.psd1"
 ConfirmExists "PowerShell\GoogleCloud\$normalizedVersion\fullclr\Google.PowerShell.dll"
 ConfirmExists "PowerShell\GoogleCloud\$normalizedVersion\coreclr\Google.PowerShell.dll"
 ConfirmExists "PowerShell\GoogleCloud\$normalizedVersion\GoogleCloud.psm1"
-ConfirmExists "PowerShell\GoogleCloud\$normalizedVersion\BootstrapCloudToolsForPowerShell.ps1"
+ConfirmExists "PowerShell\GoogleCloud\BootstrapCloudToolsForPowerShell.ps1"
```

Comment:
Should this be \GoogleCloud\$normalizedVersion\?

Is there a change something with this changes between versions?

Reply:
Can you clarify the second question?

Reply:
Can BootstrapCloudToolsForPowerShell.ps1 change between v1 1.0.0.4 and 1.0.0.5?  If so should we version this too?

Reply:
Oh I see. The BootstrapCloudToolsForPowerShell.ps1 won't change. Its only job is to launch the module.